### PR TITLE
Fix include paths

### PR DESCRIPTION
**Pull request by bogdanm**

### CODE REVIEW

**Comment by bogdanm** [comment]
@0xc0170 @bremoran

**Comment by 0xc0170** [comment]
LGTM

Hi jenkins-internal, why you are not running?

**Comment by bogdanm** [comment]
Yo, Jenkins, Martin is talking to you. Say something! 
@miklis

File: core-util/FunctionPointerBind.h [hunk 1]
```diff
@@ -22,7 +22,7 @@
 #include <stddef.h>
 #include <stdarg.h>
 #include <assert.h>
-#include "FunctionPointerBase.h"
+#include "core-util/FunctionPointerBase.h"
```
Comment (bremoran) [inline core-util/FunctionPointerBind.h]: This isn't necessary, the two files are co-located.

File: core-util/FunctionPointer.h [hunk 2]
```diff
@@ -21,8 +21,8 @@
 #include <stddef.h>
 #include <stdarg.h>
 #include <new>
-#include "FunctionPointerBase.h"
-#include "FunctionPointerBind.h"
+#include "core-util/FunctionPointerBase.h"
+#include "core-util/FunctionPointerBind.h"
```
Comment (bremoran) [inline core-util/FunctionPointer.h]: This isn't necessary, the three files are co-located.

**Comment by bremoran** [comment]
Not even sure the above comments are anything to worry about.  LGTM

**Comment by bogdanm** [comment]
Looks like the build worked, so I'll merge this unless there are any objections.

**Comment by bremoran** [comment]
LGTM

**Pull request merged**